### ISSUE TITLE
ErrorReport class shape does not match API response

### ISSUE THREAD

**Issue by akosasante** [messagebird/java-rest-api#91]
We are using the `MessageBirdClient.sendVoiceCall` method to initiate phone calls. We were able to catch exceptions when the voice call request fails. However, we also expected to be able to use the `MessageBirdException.getErrors` method to pull out the particular error code returned (not just the HTTP status code). Instead, the method returns `null`.

As per [the code here](https://github.com/messagebird/java-rest-api/blob/master/api/src/main/java/com/messagebird/exceptions/MessageBirdException.java#L34) and the status codes[ outlined in the API docs here](https://developers.messagebird.com/api#api-errors):  (eg. `25` for "not enough balance").

It looks to me like the issue is[ here in `MessageBirdServiceImpl` ](https://github.com/messagebird/java-rest-api/blob/master/api/src/main/java/com/messagebird/MessageBirdServiceImpl.java#L534) where the body of the failed HTTP request is mapped onto the ErrorReport[] class. The fields of an error I received when testing the API via cURL ("message", "code") do not match the fields of the [ErrorReport class ](https://github.com/messagebird/java-rest-api/blob/master/api/src/main/java/com/messagebird/objects/ErrorReport.java)("description", "code", "parameter"):
```
{
  "errors": [
    {
      "message": "You're not authorized to access this resource.",
      "code": 14
    }
  ]
}
```

vs 

```
public class ErrorReport {
    private Integer code;
    private String description;
    private String parameter;
...
```

I am not sure if those fields ErrorReport class are expected elsewhere or with different types of errors. The only one we were able to manually test is when we receive a 403 "unauthorized" for using the incorrect API key.

(CC: @katchengli)

**Comment by denizkilic** [comment]
Hello @akosasante,
Thanks for reporting this issue. I did a quick test. The problem caused because our APIs return back an error with different fields. For example, rest API returns code, description, parameter fields in error object but voice API returns message and code parameters. That's why it cannot parse to ErrorReport object. We will fix this problem.

**Comment by denizkilic** [comment]
Hi @akosasante,
Regarding this issue, we have just released version 3.0.8. We added the 'message' field which is not null in voice api results. Thank you and we are welcoming more issues.
